watson/watson.py

```diff
@@ -247,6 +248,13 @@ def start(self, project, tags=None):
                 )
             )
 
+        default_tags = self._default_tags(project)
+        if default_tags and not restart:
+            if tags is None:
+                tags = default_tags
+            else:
+                tags = tags + default_tags
```

Comment:
I do not think that it makes sense to remove only the duplicates between the user provided tags and the default tags.  It makes much more sens to me to just deduplicate the final list of tags.

Indeed, my solution for that creates many small lists in the process. I do not think it is much of an issue as there will never be enough tags to make it slow. Though, it can get replaced by something simpler like:

```python
def deduplicate(sequence):
    result = []
    for element in sequence:
        if element not in result:
            result.append(element)
    return result
```